packages/docs/src/components/app/index.js

```diff
@@ -196,9 +199,19 @@ const menuItems = [
 export default class Home extends Component {
   constructor() {
     super();
+
+    // Dark mode
+    const darkMode =
+      typeof localStorage !== 'undefined'
+        ? localStorage.getItem('dark-mode') === 'dark'
+        : false;
+    if (darkMode) {
+      document.body.classList.add('mdc-theme--dark');
```

Comment:
NIT: may be we can just click on the switch instead?

Reply:
Yes the switch is still there, but each time you reload the page you have to put it into dark mode again. Now it's the same as before, but it remembers the users choice

Reply:
I personally find it really annoying when page becomes white again suddenly and as it an easy fix I implemented it. What do you think?

Reply:
oh yeah what i mean is should we just `$(switch).click()` instead of doing this with classes?

Reply:
The only thing we would save is this line:
```typescript
document.body.classList.add('mdc-theme--dark');
```

And that approach feels kind of _cleaner_ to me, I don't like UI elements that do things on their own